client/src/ui/atoms/button/index.tsx

```diff
@@ -84,6 +86,23 @@ export const Button = ({
   }
 
   if (href) {
+    if (target) {
+      return (
+        <a
+          href={href}
+          target={target}
+          rel={rel}
+          className={buttonClasses}
+          id={id}
+          onClick={onClickHandler}
+          onFocus={onFocusHandler}
+          aria-label={ariaLabel}
+          title={title}
+        >
+          <span className="button-wrap">{renderContent()}</span>
+        </a>
+      );
+    }
```

Comment:
Nit: I think we should rather pass the `target` on to `InternalLink`, to avoid duplication here.

There, we could reuse the following path if a target was provided:

https://github.com/mdn/yari/blob/cc7ec93a3dac66ef6dd88b982186e6250c93ea1b/client/src/ui/atoms/internal-link/index.tsx#L9-L15